ui/src/app/components/extend-initial-credits-modal.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import { useState } from 'react';
+
+import { Profile } from 'generated/fetch';
+
+import { Button, StyledExternalLink } from 'app/components/buttons';
+import {
+  Modal,
+  ModalBody,
+  ModalFooter,
+  ModalTitle,
+} from 'app/components/modals';
+import { Spinner } from 'app/components/spinners';
+import { profileApi } from 'app/services/swagger-fetch-clients';
+import colors, { colorWithWhiteness } from 'app/styles/colors';
+import { displayDateWithoutHours } from 'app/utils/dates';
+import { notificationStore, serverConfigStore } from 'app/utils/stores';
+import { supportUrls } from 'app/utils/zendesk';
+
+import { AoU } from './text-wrappers';
+
+interface Props {
+  onClose: Function;
+  expirationDate: number;
+}
+
+const ExtensionDescription = ({ expirationDate }) => {
+  const {
+    initialCreditsValidityPeriodDays,
+    initialCreditsExpirationWarningDays,
+  } = serverConfigStore.get().config;
+  return (
+    <div>
+      Initial credits provided by the <AoU /> Research Program expire after{' '}
+      {initialCreditsValidityPeriodDays} days. You are eligible to request an
+      extension within {initialCreditsExpirationWarningDays} days of your
+      current expiration date on {displayDateWithoutHours(expirationDate)}.
```

Comment:
This language confuses me. 

You are eligible to request an extension within X days of your current expiration date on MMM DD, YYYY.

When I am reading this, it is unclear to me whether the date is the expiration date or the date on which I am eligible to request an extension.